### ISSUE TITLE
improve dependency to bputils

### ISSUE THREAD

**Issue by aodag** [beproud/bpcommons#1]
Supporting py3 with bputils is very difficult.
I think bputils split down to multiple distribution and update the few modules needed by bpcommons.

below are modules bputils used by bpcommons.
-  beproud.utils.javascript
- beproud.utils.strutils
- beproud.utils.html

```
beproud/django/commons/utils/javascript.py:from beproud.utils.javascript import escapejs_json, SafeJSONEncoder
beproud/django/commons/templatetags/string_tags.py:    from beproud.utils.strutils import abbrev
beproud/django/commons/templatetags/html_tags.py:    from beproud.utils.html import urlize
beproud/django/commons/templatetags/html_tags.py:    from beproud.utils.html import urlize
beproud/django/commons/templatetags/html_tags.py:    from beproud.utils.javascript import force_js
```

**Comment by aodag** [comment]
implementations

* https://github.com/beproud/bputils/blob/develop/beproud/utils/javascript.py
* https://github.com/beproud/bputils/blob/develop/beproud/utils/html.py
* https://github.com/beproud/bputils/blob/develop/beproud/utils/strutils.py

**Comment by shimizukawa** [comment]
Because many dependencies to 'slow maintained package' will annoy us, all depended small code/functions are now included in this project by #20.
